types/wlr_cursor.c

```diff
@@ -270,13 +276,17 @@ void wlr_cursor_absolute_to_layout_coords(struct wlr_cursor *cur,
 		double *lx, double *ly) {
 	assert(cur->state->layout);
 
-	struct wlr_box *mapping = get_mapping(cur, dev);
-	if (!mapping) {
-		mapping = wlr_output_layout_get_box(cur->state->layout, NULL);
+	struct wlr_box mapping;
+	if (!get_mapping(cur, dev, &mapping) &&
+			!wlr_output_layout_get_box(cur->state->layout, NULL, &mapping)) {
+		// no outputs in the layout
```

Comment:
dunno what to do here.

Reply:
I think setting everything to zero is sane.